### ISSUE TITLE
ErrorIs false positive

### ISSUE THREAD

**Issue by mcdoker18** [Antonboom/testifylint#31]
I have my own function with the signature that looks like `errors.Is`.  For example:

`example/lol/lol.go`:

```go
package lol

func Is(err error, _ int) bool {
	return false
}
```

`example/kek_test.go`:

```go
package kek

import (
	"errors"
	"testing"

	"github.com/Antonboom/testifylint/example/lol"
	"github.com/stretchr/testify/require"
)

func TestKek(t *testing.T) {
	err := errors.New("some text")
	require.True(t, lol.Is(err, 2))
}
```

Linter throw the following error:

```
/home/mcdoker18/opensource/Antonboom/testifylint/example/kek_test.go:13:2: error-is-as: use require.ErrorIs
```

Maybe this is a generic problem that affects other checkers as well.

**Comment by Antonboom** [comment]
@mcdoker18 fixed, thanks!

**Comment by mcdoker18** [comment]
Thank  you! I hope it will be in the next patch version of golangci-lint.
